CHANGELOG.md

```diff
@@ -7,12 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- feat(sumologicschemaprocessor): add translating attributes
+
+### Changed
+
+- feat(sumologicexporter): deprecate translating attributes ([upgrade guide][upgrade_guide_unreleased_moved_translation])
+
```

Comment:
We can add it to description as well

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/blob/main/CHANGELOG.md?plain=1#L21-L23